### PR TITLE
NP-3363: Swagger draft for one person and list of persons

### PR DESCRIPTION
**Pull request by torarnet**

### CODE REVIEW

**Review by axthosarouris — COMMENTED** [review]
Not many comments, except for what Rurik has commented already

File: docs/person-swagger.yaml [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+                  'nb': 'Gjest'
+          - id: 'https://api.dev.nva.aws.unit.no/cristin/person/567987'
+            type: 'Person'
+            identifiers:
+              - type: 'CristinIdentifier'
+                value: '567987'
+            firstName: 'Kjetil'
+            lastName: 'Nordmann'
+            affiliations:
+              # Split into Unit and Institution?
```
[the remark this answers is not in the document]
Comment (torarnet) [reply docs/person-swagger.yaml:351]: Should we add unit as a sub organization to institution? While still using type: Organization for both?

File: docs/person-swagger.yaml [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+          in: query
+          description: Search string. First name and/or last name
+          required: true
+          schema:
+            type: string
+            pattern: '.+'
+          example: 'Ola Nordmann'
+        - name: language
+          in: query
+          description: Preferred language for some fields
```
[the remark this answers is not in the document]
Comment (torarnet) [reply docs/person-swagger.yaml:31]: Should i change anything in swagger based on BCP-47 compatible?

File: docs/person-swagger.yaml [hunk 3]
```diff
@@ -0,0 +1,386 @@
...
+          description: Is active at this organizaiton
+        position:
+          $ref: '#/components/schemas/Languages'
+  examples:
+    PersonsExample:
+      value:
+        '@context': 'https://example.org/search-api-context.json'
+        id: 'https://api.dev.nva.aws.unit.no/cristin/person/?language=nb&page=1&results=5&query=Ola Nordmann'
+        size: 150
+        searchString: 'language=nb&page=1&results=5&query=Ola Nordmann'
```
Comment (brinxmat) [inline docs/person-swagger.yaml:308]: Ola%20Nordmann or Ola+Nordmann?

File: docs/person-swagger.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (brinxmat) [reply docs/person-swagger.yaml:351]: I don't think it is relevant, the distinction between an Institution and a Unit is that the former is not part of something else — if the distinction is relevant at all

**Review by brinxmat — APPROVED** [review]

**Pull request merged**